shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java

```diff
@@ -203,6 +212,60 @@ private BindingTableRule createBindingTableRule(final String bindingTableGroup)
         result.getTableRules().putAll(tableRules);
         return result;
     }
+
+    private void checkTheSameActualDataSourceNamesAndActualTableIndex(final Collection<String> bindingTableGroups, final Collection<ShardingTableRuleConfiguration> tables) {
+        if (null == bindingTableGroups || bindingTableGroups.size() <= 0) {
```

Comment:
@cheese8 Do you think `bindingTableGroups.isEmpty()` is better?

Reply:
ok